web/html/src/manager/maintenance/calendar/web-calendar.tsx

```diff
@@ -203,6 +216,39 @@ const WebCalendar = (props: WebCalendarProps) => {
     return filteredEvents.length > 0;
   }
 
+  const processEvents = (events) => {
+    // Add a color to the event cycling through the list of available colors
+    if (props.eventNames && props.eventNames.length > 0) {
+      props.eventNames.forEach((name, i) =>
+        events.filter(event => event.title === name).map(event =>
+          Object.assign(event, {color: colors[(i + colors.length) % colors.length]})
+        )
+      );
+    }
+    events.forEach(event =>
+      Object.assign(event, {
+        start: event.start.split(" ")[0],
```

Comment:
I'm guessing `event.start` is a string a-la `9:00am CEST`? I would recommend passing an ISO datetime string or a Date object and then using a-la `moment.format()` instead. This will allow us to easily accommodate for user-defined formats in the future for https://github.com/SUSE/spacewalk/issues/9049